charts/akeyless-secure-remote-access/templates/akeyless-zero-trust-portal/configmap.yaml

```diff
@@ -11,8 +11,8 @@ data:
   env-config.js: |
     window._env_ = {
         CLI_GW_DNS: {{ $.Values.apiGatewayURL | quote }},
-        AUTH_DNS: 'https://auth.akeyless.io',
-        CERT_AUTH_DNS: 'https://auth-cert.akeyless.io',
-        CLI_GW_DNS_V_2: 'https://api.akeyless.io'
+        AUTH_DNS: 'https://auth.{{ include "akeylessTenantUrl" . }}',
```

Comment:
if the options are:
```
akeyless.io
eu.akeyless.io
us.akeyless.io
```
it may should be documented somehow

Reply:
@aviram83 true,
default is "akeyless."
the other two options will be listed in a comment above